doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -773,3 +773,8 @@
     ```
 
     Have a look at the logger design document for more details on how to setup the testing logger.
+
+40. Changed the include path and namespace of several classes:
+
+    * `cxx::unique_ptr` to `memory::unique_ptr`
+        * `iceoryx_hoofs/cxx/unique_ptr.hpp` to `iceoryx_hoofs/memory/unique_ptr.hpp`
```

Comment:
With #1593 this will change to:

```suggestion
    * `cxx::unique_ptr` to `iox::unique_ptr`
        * `iceoryx_hoofs/cxx/unique_ptr.hpp` to `iox/unique_ptr.hpp`
```

Reply:
@elfenpiff @FerdinandSpitzschnueffler ^